docker/test_env/genesis_generator_helpers.go

```diff
@@ -0,0 +1,287 @@
+ETH1_FOLLOW_DISTANCE: 12
+
+SLOTS_PER_EPOCH: $SLOTS_PER_EPOCH
+
+# Validator cycle
+# ---------------------------------------------------------------
+# 2**2 (= 4)
+INACTIVITY_SCORE_BIAS: 4
+# 2**4 (= 16)
+INACTIVITY_SCORE_RECOVERY_RATE: 16
+# 2**4 * 10**9 (= 16,000,000,000) Gwei
+EJECTION_BALANCE: $EJECTION_BALANCE
+# 2**2 (= 4)
+MIN_PER_EPOCH_CHURN_LIMIT: 4
+# 2**16 (= 65,536)
+CHURN_LIMIT_QUOTIENT: 65536
+# [New in Deneb:EIP7514] 2**3 (= 8)
+MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT: $MAX_CHURN
+
+# Fork choice
+# ---------------------------------------------------------------
+# 40%
+PROPOSER_SCORE_BOOST: 40
+
+# Deposit contract
+# ---------------------------------------------------------------
+DEPOSIT_CHAIN_ID: $CHAIN_ID
+DEPOSIT_NETWORK_ID: $CHAIN_ID
+DEPOSIT_CONTRACT_ADDRESS: $DEPOSIT_CONTRACT_ADDRESS
+
+# Networking
+# ---------------------------------------------------------------
+# 10 * 2**20 (= 10485760, 10 MiB)
+GOSSIP_MAX_SIZE: 10485760
+# 2**10 (= 1024)
+MAX_REQUEST_BLOCKS: 1024
+# 2**8 (= 256)
+EPOCHS_PER_SUBNET_SUBSCRIPTION: 256
+# MIN_VALIDATOR_WITHDRAWABILITY_DELAY + CHURN_LIMIT_QUOTIENT // 2 (= 33024, ~5 months)
+MIN_EPOCHS_FOR_BLOCK_REQUESTS: 33024
+# 10 * 2**20 (=10485760, 10 MiB)
+MAX_CHUNK_SIZE: 10485760
+# 5s
+TTFB_TIMEOUT: 5
+# 10s
+RESP_TIMEOUT: 10
+ATTESTATION_PROPAGATION_SLOT_RANGE: 32
+# 500ms
+MAXIMUM_GOSSIP_CLOCK_DISPARITY: 500
+MESSAGE_DOMAIN_INVALID_SNAPPY: 0x00000000
+MESSAGE_DOMAIN_VALID_SNAPPY: 0x01000000
+# 2 subnets per node
+SUBNETS_PER_NODE: 2
+# 2**8 (= 64)
+ATTESTATION_SUBNET_COUNT: 64
+ATTESTATION_SUBNET_EXTRA_BITS: 0
+# ceillog2(ATTESTATION_SUBNET_COUNT) + ATTESTATION_SUBNET_EXTRA_BITS
+ATTESTATION_SUBNET_PREFIX_BITS: 6
+
+# Deneb
+# 2**7 (=128)
+MAX_REQUEST_BLOCKS_DENEB: 128
+# MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK
+MAX_REQUEST_BLOB_SIDECARS: 768
+# 2**12 (= 4096 epochs, ~18 days)
+MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS: 4096
+# 6
+BLOB_SIDECAR_SUBNET_COUNT: 6
+# uint64(6)
+MAX_BLOBS_PER_BLOCK: 6
+`
+
+var mnemonics = `
+- mnemonic: "${EL_AND_CL_MNEMONIC}"  # a 24 word BIP 39 mnemonic
+  count: $NUMBER_OF_VALIDATORS
+`
+
+type ExitCodeStrategy struct {
+	expectedExitCode int
+	timeout          time.Duration
+	pollInterval     time.Duration
+}
+
+func NewExitCodeStrategy() *ExitCodeStrategy {
+	return &ExitCodeStrategy{
+		expectedExitCode: 0,
+		timeout:          2 * time.Minute,
+		pollInterval:     2 * time.Second,
+	}
+}
+
+func (w *ExitCodeStrategy) WithTimeout(timeout time.Duration) *ExitCodeStrategy {
+	w.timeout = timeout
+	return w
+}
+
+func (w *ExitCodeStrategy) WithExitCode(exitCode int) *ExitCodeStrategy {
+	w.expectedExitCode = exitCode
+	return w
+}
+
+func (w *ExitCodeStrategy) WithPollInterval(pollInterval time.Duration) *ExitCodeStrategy {
+	w.pollInterval = pollInterval
+	return w
+}
+
+// WaitUntilReady implements Strategy.WaitUntilReady
+func (w *ExitCodeStrategy) WaitUntilReady(ctx context.Context, target tcwait.StrategyTarget) (err error) {
```

Comment:
Seems like we are using it just to wait for the init container to finish in "success" state or something like that?